vsm/src/vespa/vsm/vsm/docsumfilter.cpp

```diff
@@ -118,6 +118,18 @@ class RawResultHandler : public Handler {
 
 namespace vsm {
 
+FieldPath
+copyPathButFirst(const FieldPath & rhs) {
+    // skip the element that correspond to the start field value
+    FieldPath path;
+    if ( ! rhs.empty()) {
+        for (auto it = rhs.begin() + 1; it != rhs.end(); it++) {
```

Comment:
Consider `++it` instead of `it++` to avoid any implicit temporaries

Reply:
Old habits....